### ISSUE TITLE
Timeline for offline support

### ISSUE THREAD

**Issue by felix-ht** [mapbox/mapbox-maps-flutter#11]
First of all thanks for getting this out and making the source code public!

Whats the timeline for offline support? We would need that feature before we can migrate.

**Comment by Arjun544** [comment]
@felix-ht Waiting too, kindly let us know if the timeline is in near future.

**Comment by yunikkk** [comment]
Hey @felix-ht , we don't have a committed timeline for offline and other features yet. Will update the ticket as soon as it'll be decided.

**Comment by iKlsR** [comment]
@yunikkk To not open a new issue, will turn by turn be a separate repo in the future as well?

**Comment by steveneal** [comment]
Any news for this please? We're waiting on this feature in order to be able to adopt this plugin. Thanks.

**Comment by sikandernoori** [comment]
> Hey @felix-ht , we don't have a committed timeline for offline and other features yet. Will update the ticket as soon as it'll be decided.

@yunikkk, any update on this ?

**Comment by ravitejaavv** [comment]
Any update on this?

**Comment by pedrocganem** [comment]
any updates on this?

**Comment by illequipped** [comment]
Flutter is growing. It would be great to get this implemented.

**Comment by maios** [comment]
Hi all, we have added our first support for Offline map https://github.com/mapbox/mapbox-maps-flutter/pull/548
It will be included in our MapboxMaps Flutter v2.1.0-beta.1 scheduled this week. Of course there is always room for improvement, so please try it out and help us make it better by reportng issues. Thank you for supporting this project <3

**Comment by dimonkomaron** [comment]
Hi, @maios! Are there any plans to expose lifecycle events for downloads, such as cancel, resume, or restart?

Currently, when I try to pause a download by removing a region from the tileStore, it doesn't seem to stop the ongoing download. Additionally, after the download completes, I encounter an error stating that the region does not exist.

**Comment by O-Hannonen** [comment]
Offline support has already landed, so closing this issue! 

@dimonkomaron could you open a new issue about the cancellation for better visibility?
